core-errors/src/simple.rs

```diff
@@ -341,6 +342,37 @@ impl SimpleProgramCreationError {
     }
 }
 
+#[repr(u8)]
+#[derive(
+    Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Default, derive_more::Display,
+)]
+#[cfg_attr(feature = "codec", derive(Encode, Decode, TypeInfo, Sequence), codec(crate = scale), allow(clippy::unnecessary_cast))]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+/// Simplified error occurred because of actor inactivity.
+pub enum SimpleInactiveActorError {
```

Comment:
program terminated / not found: any init-related cases of inactivity we've discussed?